src/uwtools/forecast_driver.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+#pylint: disable=unused-import, unused-variable, unused-argument
+'''
+This file contains the scaffolding for the forecast driver. 
+It is a python script that will be called by the user to run the 
+forecast. It will call the other tools in the uwtools package 
+to do the work. It will also be responsible for setting up the 
+working directory and moving files around.
+'''
+
+import argparse
+import inspect
+import logging
+import os
+import sys
+
+from uwtools.j2template import J2Template
+from uwtools import config
+
+def parse_args(argv):
+
+    '''
+    Function maintains the arguments accepted by this script. Please see
+    Python's argparse documentation for more information about settings
+     of each argument.
+    '''
+
+def create_model_config():
```

Comment:
This and all the following functions should probably go in a Forecast class. Recall, that we have a description of a component driver design [here](https://github.com/ufs-community/workflow-tools/wiki/Migrating-Production-Workflows-to-the-Unified-Workflow-using-the-Strangler-Fig-Pattern#component-drivers).

Again, we'll likely want a command line interface that handles user input and such, but remember that here, the goal is capturing the methods, inputs, and outputs of the _Class_ and its methods. The command line interface isn't quite as big a deal right now, as it will likely consist of only a couple of meaningful calls like:

```
def main():
    args = parse_args()
    forecast = Forecast(args)
    forecast.run()
```

Reply:
Still fixing main() unfortunately!